packages/perseus/src/widgets/numeric-input/numeric-input-validator.ts

```diff
@@ -107,62 +108,48 @@ function numericInputValidator(
     // We may have received TeX; try to parse it before grading.
     // If `currentValue` is not TeX, this should be a no-op.
     const currentValue = ParseTex(userInput.currentValue);
-    const correctAnswers = rubric.answers.filter(
-        (answer) => answer.status === "correct",
-    );
-
-    const normalizedAnswerExpected = correctAnswers.every(
-        (answer) => Math.abs(answer.value) <= 1,
-    );
-
-    // Look through all correct answers for one that matches either
-    // precisely or approximately and return the appropriate message:
-    // - if precise, return the message that the answer came with
-    // - if it needs to be simplified, etc., show that message
-    let result = correctAnswers
+
+    const normalizedAnswerExpected = rubric.answers
+        .filter((answer) => answer.status === "correct")
+        .every((answer) => Math.abs(answer.value) <= 1);
+
+    // The coefficient is an attribute of the widget
+    let localValue: string | number = currentValue;
```

Comment:
Ignorable Nit: We could possibly explain a little more about the coefficient here, but perhaps that's best handled in the widget options.